1-development/list_posts_by_rating/app/services/post/downvote_create_service.rb

```diff
@@ -0,0 +1,7 @@
+class Post
+  class DownvoteCreateService
+    def self.call(post)
```

Comment:
Why create multiple services and not use methods from the same service?

Reply:
Leave each of the services isolated. If we want to have a different rule for each of the operations it would make it easier.